package.json

```diff
@@ -4,6 +4,20 @@
     "examples/*",
     "packages/*"
   ],
+  "exports": {
+    ".": {
+      "types": "./packages/redux-saga/index.d.ts",
+      "browser": "./packages/redux-saga/dist/redux-saga-core.esmodules-browsers.js",
+      "umd": "./packages/redux-saga/dist/redux-saga.umd.js",
+      "require": "./pacages/redux-saga/dist/redux-saga-core-npm-proxy.cjs.js"
+    },
+    "./effects": {
+      "types": "./packages/redux-saga/effects.d.ts",
+      "browser": "./packages/redux-saga/dist/redux-saga-effects.esmodules-browsers.js",
```

Comment:
It might also be useful to add a `development` (.js file) and `production` (min.js file) condition here for browsers?

Reply:
https://nodejs.org/api/packages.html#community-conditions-definitions

It seems reasonable to provide both and we already have the builds so I updated the PR to include them.

Reply:
That would be great if you could.  I tested esm and cjs via esbuild and everything appeared to work correctly.

Reply:
It all looks good to me from over here, too!